src/components/reusable/notification/notification.ts

```diff
@@ -0,0 +1,141 @@
+import { LitElement, html } from 'lit';
+import { customElement, property, state } from 'lit/decorators.js';
+
+import NotificationScss from './notification.scss';
+import '@kyndryl-design-system/shidoka-foundation/components/card';
+import '../tag';
+
+/**
+ * Notification
+ * @slot action-slot - Slot for menu.
+ * @slot notification-body-slot - Slot for notification message.
+ * @fires on-notification-click - Emit event for clickable notification.
+ */
+
+@customElement('kyn-notification')
+export class Notification extends LitElement {
+  static override styles = NotificationScss;
+
+  /** Notification Title (Required). */
+  @property({ type: String })
+  notificationTitle = '';
+
+  /** Notification subtitle.(optional) */
+  @property({ type: String })
+  notificationSubtitle = '';
+
+  /** Timestamp of notification. */
+  @property({ type: String })
+  timeStamp = '';
+
+  /** Card href link */
+  @property({ type: String })
+  href = '';
+
+  /** Notification status tag type. `'default'`, `'info'`, `'warning'`, `'success'` & `'error'` */
+  @property({ type: String })
+  tagStatus = 'default';
+
+  /** Notification type. `'normal'` and `'clickable'`. clickable type can be use inside notification panel */
+  @property({ type: String })
+  type = 'normal';
+
+  /** Notification tag label. Insert text of tag according to notification status type. (Requires `tagStatus` other than default) */
+  @property({ type: String })
+  tagLabel = '';
+
+  /** Set tagColor based on provided tagStatus.
+   * @ignore
+   */
+  @state()
+  tagColor: any = '';
+
+  override render() {
+    return html`
+      ${this.type === 'clickable'
+        ? html`<kd-card
+            type=${this.type}
+            href=${this.href}
+            target="_blank"
```

Comment:
`target` should not be hard-coded to new tab. 
Also note that `target` and `rel` are kinda tied together, `rel="noopener"` is expected when `target="_blank"`

Reply:
oh yes. I think I forgot to provide `rel="noopener"` should I add it or I remove `target="_blank"`

Reply:
I think add it, so both values can be configured.